contracts/lib.rs

```diff
@@ -2285,7 +2277,7 @@ pub mod prosopo {
             let provider_account = AccountId::from([0x2; 32]);
             // initialise the contract
             let mut contract = Prosopo::default(operator_account, STAKE_DEFAULT, STAKE_DEFAULT);
-            assert_eq!(0, contract.get_provider_balance(provider_account));
+            contract.get_provider_balance(provider_account).unwrap_err();
```

Comment:
What is the error? Should this be returning `ProviderDoesNotExist`?

Reply:
The `unwrap_err()` is expecting an error, unspecific to which type of error. It panics when the result is Ok, which it will not be because the provider_account has not been loaded into the contract